module1/homework/removeVowels/vowel.cpp

```diff
@@ -0,0 +1,19 @@
+#include "vowel.hpp"
+
+#include <algorithm>
+#include <array>
+
+void removeVowels(std::vector<std::string>& vectorOfSentences) {
+    std::array<char, 12> vowelsArr{'a', 'e', 'i', 'u', 'y', 'o', 'A', 'E', 'I', 'U', 'Y', 'O'};
```

Comment:
I think you can make it constrexpr and I would consider moving it to the header.

Reply:
I tried to do it, but the compiler gave me this error.
![static](https://user-images.githubusercontent.com/62703601/87147390-d7dd7600-c2ac-11ea-83b4-644c6dc3141a.png)

Reply:
I think it's because you use &vowelsArr(no wonder why passing constexpr by reference causes error). Would it work if you put vowelsArr in lamba instead?